src/js/components/TabPanesComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var AppListComponent = require("../components/AppListComponent");
+var DeploymentsListComponent =
+  require("../components/DeploymentsListComponent");
+var TabPaneComponent = require("../components/TabPaneComponent");
+var TogglableTabsComponent = require("../components/TogglableTabsComponent");
+
+var tabs = [
+  {id: "/apps", text: "Apps"},
+  {id: "/deployments", text: "Deployments"}
+];
+
+var TabPanesComponent = React.createClass({
+  displayName: "TabPanesComponent",
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    return {
+      tabId: this.getTabId()
+    };
+  },
+
+  componentWillReceiveProps: function () {
+    this.setState({
+      tabId: this.getTabId()
+    });
+  },
+
+  getTabId: function () {
+    var path = this.context.router.getCurrentPathname();
+
+    if (tabs.find(function (tab) {
+      return tab.id === path;
+    })) {
```

Comment:
Can we write the `tabs.find(...)` on it's own line and assign it to a variable, and use that in the `if`.